src/main/java/seedu/address/logic/commands/editcommand/EditMentorCommand.java

```diff
@@ -53,10 +91,86 @@ private Mentor createEditedMentor(Mentor mentorToEdit, EditMentorDescriptor edit
      */
     public static class EditMentorDescriptor extends EditEntityDescriptor {
 
-        /*
-         * Implement the remaining attributes
+        private Phone phone;
```

Comment:
btw shouldn't the editMentorDescriptor also have a field for name in case we want to change his name?

Reply:
Same applies to the editParticipant and editTeam. (I hope I'm not missing anything here lol)

Reply:
Oh EditCommand has EditEntityDescriptor that has name and id. So all other EditXDescriptor has get/setName. But on the second thought, I should go and erase getter and setter for ID haha

Reply:
Great I did miss something sighh. Awesome then all good from me 👍 !